benchmarks/ios/animation-from-template/shared/build.gradle.kts

```diff
@@ -24,7 +24,6 @@ kotlin {
             baseName = "shared"
             isStatic = true
         }
-        extraSpecAttributes["resources"] = "['src/commonMain/resources/**', 'src/iosMain/resources/**']"
```

Comment:
The changes in the samples and in the benchmarks should not be a part of this commit until we have a stable compose build that contains new resource management logic. We may create a separate PR for them that will be merged once we have such a version. And I would recommend to migrate all of them to regular framework and add a separate sample with CocoaPods usage.